From ece94b7de3f051a4a417599f1cbf1e5cb6bfbb74 Mon Sep 17 00:00:00 2001
From: xhe <xw897002528@gmail.com>
Date: Fri, 28 Dec 2018 22:43:45 +0800
Subject: [PATCH 39/48] poparser: not check overflow for msgid_plural it's
 allowed to have nplurals=1, but with msgid_plural and msgstr[0]

---
 src/poparser.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/poparser.c b/src/poparser.c
index 46aaa9f..d7aafb1 100644
--- a/src/poparser.c
+++ b/src/poparser.c
@@ -289,10 +289,6 @@ enum po_error poparser_feed_line(struct po_parser *p, char* in, size_t in_len) {
 			if (!msg->id_len || p->strcnt)
 				return -po_invalid_entry;
 
-			if (p->strict && p->hdr.nplurals < 2) {
-				return -po_plurals_overflow;
-			}
-
 			if (p->stage == ps_parse) {
 				if (msg->plural == NULL) {
 					return -po_internal;
-- 
2.25.0

